atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/cache/CacheEntry.java

```diff
@@ -45,12 +45,16 @@ static CacheEntry unlocked(CacheValue value) {
                 .build();
     }
 
+    default boolean isUnlocked() {
+        return status().isUnlocked();
+    }
+
     enum Status {
         LOCKED,
         UNLOCKED;
 
-        public boolean isUnlocked() {
-            return this == Status.UNLOCKED;
+        boolean isUnlocked() {
+            return this.equals(Status.UNLOCKED);
```

Comment:
I know the underlying equality check for an enum is `==` but it just feels wrong in hindsight.